cg/apps/mip/parse_qcmetrics.py

```diff
@@ -29,28 +27,24 @@ def set_collecthsmetrics_metrics(file_metrics: dict, sample_data: dict) -> dict:
     }
     sample_data["gc_dropout"] = float(hs_metrics["GC_DROPOUT"])
     sample_data["target_coverage"] = float(hs_metrics["MEAN_TARGET_COVERAGE"])
-    return sample_data
 
 
 def set_collectmultiplemetrics_metrics(file_metrics: dict, sample_data: dict) -> dict:
     """Set collectmultiplemetrics metrics"""
     mm_metrics = file_metrics["collectmultiplemetrics"]["header"]["pair"]
     sample_data["strand_balance"] = float(mm_metrics["STRAND_BALANCE"])
-    return sample_data
 
 
 def set_collectmultiplemetricsinsertsize_metrics(file_metrics: dict, sample_data: dict) -> dict:
     """Set collectmultiplemetricsinsertsize metrics"""
     mm_insert_metrics = file_metrics["collectmultiplemetricsinsertsize"]["header"]["data"]
     sample_data["median_insert_size"] = int(mm_insert_metrics["MEDIAN_INSERT_SIZE"])
     sample_data["insert_size_standard_deviation"] = float(mm_insert_metrics["STANDARD_DEVIATION"])
-    return sample_data
 
 
 def set_markduplicates_metrics(file_metrics: dict, sample_data: dict) -> dict:
     """Set markduplicates metrics"""
     sample_data["duplicates"] = float(file_metrics["markduplicates"]["fraction_duplicates"])
-    return sample_data
 
 
 def get_sample_metrics(sample_metrics: dict, sample_data: dict) -> dict:
```

Comment:
I would still like to get some clearifying comments for this function. What is the argument `sample_metrics`? looks like in the code that it is `Dict[str, List[str]]` for each value there is a list of tools? If that is the case it looks like we will call the functions over and over again which should not be necessary. 

What is `file_metrics`?